.github/CODEOWNERS

```
@@ -3,7 +3,9 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj
+
+
+* @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj @rapidray12 @schmelz21 @cduhn17 @aloftus23 @Matthew-Grayson @nickviola
```

Remove owners who are not involved in this repo and alphabetize those who are:
```suggestion
* @aloftus23 @cduhn17 @Matthew-Grayson @nickviola @rapidray12 @schmelz21
```